jsonmerge/strategies.py

```diff
@@ -230,8 +231,14 @@ def merge(self, walk, base, head, schema, meta, **kwargs):
                     p = schema.get('additionalProperties')
                     if not p.is_undef():
                         subschema = p
-
-            base.val[k] = walk.descend(subschema, base.get(k), v, meta).val
+            both_def = not (base.get(k).is_undef() or head.get(k).is_undef())
+            if renameConflicts and both_def and base.get(k) != head.get(k):
+                na = JSONValue(undef=True)
+                base.val[k] = walk.descend(subschema, na, base.get(k), meta, validate=False).val
```

Comment:
If you don't want to actually merge anything lower in the hierarchy, it would be better to just copy the value instead of doing a `descend()`. See how it's done in the `Version` strategy, for example.